Playgrounds/2b_Maybe_Type_Monad.playground/contents.swift

```diff
@@ -108,13 +108,13 @@ The use of *switch statements* would add verbosity and remove clarity.
 
 Notice that the *pure* function is used to lift the return value into the **Maybe** type.
 */
-func livingSpace(person: Maybe<Person>) -> Maybe<Int> {
+func livingSpace(_ person: Maybe<Person>) -> Maybe<Int> {
     return person >>= { $0.residence }
```

Comment:
unfortunately this throws a compile error in Xcode 8 beta 4:
`error: type of expression is ambiguous without more context`